section2/exercises/methods.rb

```diff
@@ -12,21 +12,30 @@ def print_name
 
 # Write a method that takes a name as an argument and prints it:
 def print_name(name)
-  # YOUR CODE HERE
+  p "Albus Dumbledore"
 end
 
 print_name("Albus Dumbledore")
 
-# Write a method that takes in 2 numbers as arguments and prints 
+# Write a method that takes in 2 numbers as arguments and prints
 # their sum. Then call your method three times with different arguments passed in:
-# YOUR CODE HERE
+two_nums = [182, 444]
+  p two_nums.sum
+  p two_nums.unshift (341)
+  p two_nums.push (6)
+  p two_nums.pop
 
 
-# Write a method that takes in two strings as arguments and prints 
-# a concatenation of those two strings. Example: The arguments could be 
-# (man, woman) and the end result might output: "When Harry Met Sally".  
-# Then call your method three times with different arguments passed in. 
+# Write a method that takes in two strings as arguments and prints
+# a concatenation of those two strings. Example: The arguments could be
+# (man, woman) and the end result might output: "When Harry Met Sally".
+# Then call your method three times with different arguments passed in.
 
+sentence = ["Felt needling", "is cool."]
```

Comment:
See above- how could we define a method to accomplish the same thing that you figured out with Array methods?